pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -240,7 +240,8 @@ EOF
     declare -A keysSeen
 
     for registry in ${toString (builtins.attrNames extraRegistries)}; do
-      cat >> $out/.cargo/config <<EOF
+      cat >> $out/.cargo/config.toml <<EOF
+      ln -s .cargo/config.toml .cargo/config
```

Comment:
I think this may fail if it is run multiple times. Can we do the `ln` in only one place instead?